README.md

```diff
@@ -13,23 +13,69 @@ We are accepting bug reports (please report with as much detail as possible). Fe
 - A desktop platform with the [.NET Core SDK 2.2](https://www.microsoft.com/net/learn/get-started) or higher installed.
 - When working with the codebase, we recommend using an IDE with intellisense and syntax highlighting, such as [Visual Studio Community Edition](https://www.visualstudio.com/) (Windows), [Visual Studio Code](https://code.visualstudio.com/) (with the C# plugin installed) or [Jetbrains Rider](https://www.jetbrains.com/rider/) (commercial).
 
-# Building and running
+# Running osu!
 
-If you are not interested in developing the game, please head over to the [releases](https://github.com/ppy/osu/releases) to download a precompiled build with automatic updating enabled (download and run the install executable for your platform).
+## Releases
 
-Clone the repository including submodules
+If you are not interested in developing the game, please head over to the [releases](https://github.com/ppy/osu/releases) to download a precompiled build with automatic updating enabled.
 
-`git clone --recurse-submodules https://github.com/ppy/osu`
+- Windows 64 bit users should download and run `install.exe`.
+- MacOS users should download and run `osu.app.zip`.
+- iOS users can join the [TestFlight beta program](https://t.co/xQJmHkfC18).
 
-Build and run
+There is currently no release for Windows 32 bit, Linux, or any other platform. If you are not running Windows 64 bit or MacOS, you should build osu! from the source code (see below).
```

Comment:
This is poorly worded and makes it sound as if there are no releases whatsoever.